lib/src/main/kotlin/com/swmansion/starknet/crypto/HashMethod.kt

```diff
@@ -0,0 +1,18 @@
+package com.swmansion.starknet.crypto
+
+import com.swmansion.starknet.data.types.Felt
+
+enum class HashMethod {
+    PEDERSEN {
+        override fun hash(first: Felt, second: Felt): Felt {
+            return StarknetCurve.pedersen(first, second)
+        }
+    },
+    POSEIDON {
+        override fun hash(first: Felt, second: Felt): Felt {
```

Comment:
Won't we need it to be able to calculate hashes on bigger number of arguments? (pedersen as well. this is probably a nitpick, since this is more of a future-problem, probably)

Reply:
Yes, it is gonna be needed and will be added in #428. Just wanted to keep this PR focused on MerkleTree changes.